CHANGES.rst

```diff
@@ -58,6 +59,10 @@ Imviz
 Mosviz
 ^^^^^^
 
+- Fix typo where zoom Dec retrieval used RA instead. [#1992]
```

Comment:
#1992 actually fixes an unreleased bug, so maybe we don't even need it!
```suggestion
```

Reply:
You know, that's a fair point!